src/features/upload-file/upload-file.feature.ts

```diff
@@ -136,20 +211,27 @@ const uploadFileFeature = (config: Config): FeatureType => {
           ),
         },
       },
-      [properties.bucket]: { isVisible: false },
```

Comment:
removed this one because I think that this shouldn't be the responsibility of this feature. It should handle upload but visibilities of fields should be handled by the user.

Hiding them makes lots of sense but I think that users won't expect that from this feature.
What do you think?